--- macemu/SheepShaver/src/include/clip.h	2026-01-12 19:02:49.437794776 +0900
+++ macemu_patches/Clipboard_Patch_Sheepshaver/patches/clip.h	2026-01-11 19:49:43.609459054 +0900
@@ -28,4 +28,9 @@
 extern void PutScrap(uint32 type, void *scrap, int32 length);
 extern void GetScrap(void **handle, uint32 type, int32 offset);
 
+extern bool ClipboardCheck(void);
+extern int32 ClipboardGetSize(uint32 type);
+extern int32 ClipboardGetData(uint32 type, void *buffer, int32 size);
+extern void ClipboardPutData(uint32 type, void *data, int32 size);
+
 #endif
